src/main/java/uk/gov/justice/hmpps/prison/api/model/ImageDetail.java

```diff
@@ -36,6 +37,14 @@ public class ImageDetail {
     @NotNull
     private LocalDate captureDate;
 
+    @Schema(description = "Date and time of image capture", example = "2008-08-27 01:01:01")
+    @NotNull
+    private LocalDateTime captureDateTime;
+
+    @Schema(description = "Date and time of image creation", example = "2008-08-27 01:01:01")
```

Comment:
Why do you need this data? It is never exposed in NOMIS to the end user so seems very odd that it is now exposed.  Image creation is an odd term - would be better to call it image upload.

Reply:
The requirement isn't to expose this in NOMIS.
We need this information in the HMPPS Integration API. Our consumer (MAPPS) are interested in this data to display in their system.

Reply:
We're not going to expose the create date, capture date will be fine